scripts/fdbshow

```diff
@@ -44,7 +44,7 @@ class FdbShow(object):
         self.if_name_map, \
         self.if_oid_map = port_util.get_interface_oid_map(self.db)
         self.if_br_oid_map = port_util.get_bridge_port_map(self.db)
-        self.fetch_fdb_data()
+        self.fetch_state_fdb_data()
         return
 
     def fetch_fdb_data(self):
```

Comment:
I see large code duplicated in nbrshow, could you also improve it by reusing code?

Reply:
We did not use this command 'nbrshow'. But I can take a look later to see if it could be improved.